app/src/main/java/org/piwigo/ui/login/LoginViewModel.java

```diff
@@ -75,6 +75,7 @@
     private final UserManager userManager;
     private Account account = null;
 
+    boolean unitTesting = false;
```

Comment:
I don't think it is a good idea to add such code just for testing. The test coverage is so low anyhow, such that I would even claim it is better to remove the test than to add such a test variable. Thee optimal solution for sure would be to adjust the test such that it has to correct expectation..'